docs/python/conf.py

```diff
@@ -0,0 +1,103 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# pylint: disable=C0103
+
+# -*- coding: utf-8 -*-
+#
+# Configuration file for the Sphinx documentation builder.
+
+import os
+import shutil  # noqa: F401
+import sys
+
+# import recommonmark
+
+# -- Project information -----------------------------------------------------
+
+project = "ONNXRuntime-Extensions Python API"
+copyright = "2018-2023, Microsoft"
```

Comment:
Can we programmatically insert the current year instead of hardcoding 2023?

Reply:
done!